src/assets/script.js

```diff
@@ -5,22 +5,29 @@
 /* Cookie consent banner customization */
 (function () {
   window.addEventListener('tac.root_available', function (e) {
-    document.getElementById('tarteaucitronPersonalize').classList.add('btn', 'btn-primary', 'btn-inverse', 'ml-2')
-    document.getElementById('tarteaucitronAllAllowed').classList.add('btn', 'btn-primary', 'btn-inverse', 'ml-2')
-    
-    document.getElementById('tarteaucitronCloseAlert').classList.add('btn', 'btn-inverse', 'btn-info', 'ml-2')
-    document.getElementById('tarteaucitronAllDenied').classList.add('btn', 'btn-inverse', 'btn-info', 'ml-2')
+
+    document.querySelectorAll('#tarteaucitronRoot button').forEach(function (button) {
+      if (button.classList.contains('catToggleBtn') || button.id === 'tarteaucitronClosePanel') {
+        return
+      }
+
+      if (button.classList.contains('tarteaucitronAllow')) {
+        button.classList.add('btn', 'btn-primary', 'btn-inverse', 'ml-2')
+      } else {
+        button.classList.add('btn', 'btn-inverse', 'btn-info', 'ml-2')
```

Comment:
Setting `.btn-inverse` as the second name would improve consistency thus legibility ;)

Reply:
Done